### ISSUE TITLE
Update Mantid Basic Course

### ISSUE THREAD

**Issue by rbauststfc** [mantidproject/mantid#35032]
The following updates to the [Mantid Basic Course](http://docs.mantidproject.org/tutorials/mantid_basic_course/index.html#mantid-basic-course) were found to be needed during manual testing under issue #34955:

- [x] [04_displaying_2D_data](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/loading_and_displaying_data/04_displaying_2D_data.html#displaying-2d-data) - does not have the correct image or any information relating to the cutting tool added in PR #33765. _After reviewing the information provided on the page, it seems sufficient just to update the screenshots._

- [x] [06_formatting_plots.html](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/loading_and_displaying_data/06_formatting_plots.html#formatting-plots) - plot toolbar graphic needs updating, it does not include the new hide button introduced in PR #34614.

- [x] [1DPlotsHelp.html#basic-1d-plots](https://docs.mantidproject.org/nightly/plotting/1DPlotsHelp.html#basic-1d-plots) - all the script examples on this page are out of date (running them would give error "Legend has no attribute draggable"). It would be good to generate up to date examples from Mantid.

- [x] [03_workspaces.html#correct-binning](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/algorithms_workspaces_and_histories/03_workspaces.html#correct-binning) - plot generated from following the example gives [this colourfill](https://github.com/mantidproject/mantid/issues/34955#issuecomment-1385553972). Either need to update the screenshot in the course or correct the steps to allow users to produce a matching result.

- [x] [03_investigating_data.html#the-toolbar](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/03_investigating_data.html#the-toolbar) - buttons on this page need updating.

- [x] [03_investigating_data.html#selecting-arbitrary-sets-of-detectors](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/03_investigating_data.html#selecting-arbitrary-sets-of-detectors) - button positions have changed so the reference to positions here could do with being updated.

- [x] [03_investigating_data.html#picking-a-single-detector](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/03_investigating_data.html#picking-a-single-detector) - following the steps exactly as written causes the plotted curve to disappear as soon as the mouse moves off the detector. We should make it clear that the necessary steps are to click the `Add a single crystal peak` button and then click a detector in the instrument view (not just mouse over one) before moving to zoom in to the mini-plot.

- [x] [03_investigating_data.html#picking-a-tube](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/03_investigating_data.html#picking-a-tube) - similar to the above, except the necessary steps seem to be to click the tube selection button, click on a detector and then keep holding the mouse button down until the cursor is over the mini-plot. That way the plot's right click menu can then be used without the curve disappearing.

- [x] ~[04_masking_and_grouping.html#region-of-interest](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/04_masking_and_grouping.html#region-of-interest) - the last paragraph of this section looks like it should actually be under the Masking section above.~ On a second look this actually does seem to make sense, so it's simply a small typo that needs correcting here.

- [x] [06_sample_logs.html#basic-inspection](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/06_sample_logs.html#basic-inspection) - screenshot is of `BSS_11841_event` rather than `CNCS_7860_event`.

- [x] [06_sample_logs.html#algorithms](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/06_sample_logs.html#algorithms) - the paragraph underneath the screenshot in this section that begins:
> Look at the effect this filter has on the Temperature Sample Log and...

should say SampleTemp rather then Temperature Sample.

- [ ] [06_sample_logs.html#algorithms](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/06_sample_logs.html#algorithms) - underneath the paragraph described above, the explanation for the 0 relative time start is confusing and the plot that should appear under this paragraph does not appear, it just incorrectly displays `(Source Code)`. _This one should be handled as a separate issue as building locally renders the plots, so perhaps there's an issue with the Jenkins build - see #35214._

- [x] [07_invalid_sample_logs.html](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/07_invalid_sample_logs.html) - the dataset `ENGINX00228061_log_alarm_data.nxs` referenced on this page should be added to the TrainingCourseData.

- [x] [02_complex_case.html#functions-properties-setting-up-a-fit-model](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/fitting_data/02_complex_case.html#functions-properties-setting-up-a-fit-model) - the following paragraph is confusing as it mentions dragging on the plot after having set a FlatBackground, but this info should really be for the peak lines:
> Note that two fit functions have appeared in the Fit Property Browser called f0-Gaussian and f1-FlatBackground. Click on the triangles beside these functions to reveal their fit parameters. Just as with the StartX/EndX values, dragging on the plot and changing these values update each other.

- [x] [02_complex_case.html#tying-and-constraining-fit-parameters](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/fitting_data/02_complex_case.html#tying-and-constraining-fit-parameters) - the `Tie > to Function` option works fine in the case of 2 peaks rather than 1 peak plus 1 background. The docs should be changed to make it clearer that the user should use `Tie > Custom Tie` for the examples it gives.

**Comment by rbauststfc** [comment]
@DanielMurphy22, I think this item on the list of updates was referring to some data that you have:

[07_invalid_sample_logs.html](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/07_invalid_sample_logs.html) - the dataset ENGINX00228061_log_alarm_data.nxs referenced on this page should be added to the TrainingCourseData.

I'm not sure of the process for adding data to the training course, do you know how this works at all?

**Comment by DanielMurphy22** [comment]
I don't think I can directly do it. I think you need to download the current TrainingCourseData from our Downloads page and unzip it. Then add in the new file, rezip and it needs to be uploaded to the same place on sourceforge. I'm not sure I have access to upload this, I think I had to ask someone else in the team to upload it for me: https://sourceforge.net/projects/mantid/files/Sample%20Data/TrainingCourseData.zip/download

**Comment by rbauststfc** [comment]
Great, thanks, that's all really useful to know. I'm happy to sort out the new zip and arrange for it to be uploaded (plus check about adding all the sub-team leads to the admin list) if you can send me the data (or let me know where I can find it)?

**Comment by DanielMurphy22** [comment]
Great, thanks! I'll send it on Slack!

**Comment by DanielMurphy22** [comment]
For the second bullet point, the toolbar images are held in a powerpoint file here: https://github.com/mantidproject/documents/tree/main/Images/Images_for_Docs
I have just updated that document when needed and screenshotted the new toolbar to get a .png that looks good. As far as I remember the last time I had to take 2 screenshots and merge them as the toolbar has gotten quite long. Tell me if you'd rather I did this!
